backend/remote-state/s3/backend_test.go

```diff
@@ -29,16 +29,12 @@ func TestBackend_impl(t *testing.T) {
 }
 
 func TestBackendConfig(t *testing.T) {
-	// This test just instantiates the client. Shouldn't make any actual
-	// requests nor incur any costs.
-
+	testACC(t)
```

Comment:
This was already the exception to the backend tests, because previously it only did a config check (the code was inherited from the old remote state implementation). 

The rest of the backend tests already require `TF_ACC` to test any functionality.